pynest/nest/lib/hl_api_helper.py

```diff
@@ -501,15 +501,16 @@ def show_help_with_pager(hlpobj, pager=None):
         return
 
     try:
-        subprocess.check_call([pager, objf])
+        pagerl = pager.split(" ")
```

Comment:
do we want to use shlex.split here? That would also handle spaces and quotes correctly, etc.
```suggestion
        import shlex
        pagerl = shlex.split(pager)
```

Reply:
any opinion on this, anyone?

Reply:
@terhorstd That looks good. I will be trying it out.

Reply:
@terhorstd  Works for me. Changed it.